.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
+    "unicorn/explicit-length-check": [2, { "non-zero": "not-equal" }],
+    "unicorn/no-process-exit": 0,
+    "unicorn/no-null": 0,
+    "unicorn/prevent-abbreviations": 0,
+    "unicorn/number-literal-case": 0,
+    "unicorn/expiring-todo-comments": 0,
+    "unicorn/no-fn-reference-in-iterator": 0,
+    "unicorn/no-unreadable-array-destructuring": 0,
+    "unicorn/import-style": 0,
```

Comment:
Another pretty nice idea. [See rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/import-style.md). However, this might require too much refactoring at the very moment. 
Might be worth considering for the future though?

Reply:
How does it work for dependencies with only a default export?

Reply:
I think it might try to detect whether that's the case, although I'm not 100% sure.
One thing where it could be useful in terms of consistency is for Node.js core modules. Some people prefer using the default export, some prefer the named export.

Regardless, this would require too much code change for this single PR, which is why the rule is disabled.